start-k8s.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Is there a script to shutdown the Kubernetes setup? If not do we require one?

Reply:
There's no script to shutdown at the moment. 
JasmineGraph's `shdn` command will shutdown the workers. But resources will remain. (volume mounts)

Reply:
Ok